pysmt/formula.py

```diff
@@ -897,14 +897,66 @@ def BVRepeat(self, formula, count=1):
         for _ in xrange(count-1):
             res = self.BVConcat(res, formula)
         return res
-
-    def Length(self, formula):
-        """ Creates an expression of the form:
-            Length formula
+    
+    def str_length(self, formula):
```

Comment:
For consistency, all methods of Formula should be CamelCase. This should be ```StrLength```. the same applies to all other methods.